Notation.Plugin.AzureKeyVault/Certificate/CertificateBundle.cs

```diff
@@ -0,0 +1,40 @@
+using System.Security.Cryptography.X509Certificates;
+
+namespace Notation.Plugin.AzureKeyVault.Certificate
+{
+    /// <summary>
+    /// Helper class to create a certificate bundle from a PEM file.
+    /// </summary>
+    class CertificateBundle
+    {
+        /// <summary>
+        /// Create a certificate bundle from a PEM file.
+        /// </summary>
+        public static X509Certificate2Collection Create(string pemFilePath)
+        {
+            // Split the PEM file into certificates.
+            string pemContent = File.ReadAllText(pemFilePath);
+            string[] pemCertificates = pemContent.Split("-----END CERTIFICATE-----", StringSplitOptions.RemoveEmptyEntries & StringSplitOptions.TrimEntries);
+
+            // Add the certificates to the bundle.
+            var certs = pemCertificates.Select(x => ConvertPemToDer(x))
+                                       .Where(x => x != null && x.Length > 0)
+                                       .Select(x => new X509Certificate2(x));
+            return new X509Certificate2Collection(certs.ToArray());
+        }
+
+        /// <summary>
+        /// Convert PEM to DER. It removes the header and footer of the PEM 
+        /// file, merges multiple lines into one and decodes the base64 string.
+        /// </summary>
+        private static byte[] ConvertPemToDer(string pem)
+        {
+            // Remove the header and footer of the PEM file.
+            var lines = pem.Split('\n', StringSplitOptions.RemoveEmptyEntries & StringSplitOptions.TrimEntries)
+                           .Where(x => !x.StartsWith("-----"));
+
+            // Merge multiple lines into one.
+            return Convert.FromBase64String(String.Join("", lines));
```

Comment:
I'm not very sure, looks like we are manually splitting/changing/joining a PEM file here, which technically works, but bring me some hesitations.
Is there any official/formal method that we can use directly to convert a PEM to DER?

Reply:
Yes, I did it manaully because I didn't find a way to do it without introducing a third party library. I will keep finding the built-in counterpart. If you have suggested package, please let me know.

Reply:
Updated to use `X509Certificate2Collection.ImportFromPemFile` which is suggested by Shiwei.